<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insertMtdDlg</class>
 <widget class="QDialog" name="insertMtdDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insert metadata information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="habitatLabel">
         <property name="text">
          <string>Metadata tag</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="metadataEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Metadata Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="metadataTypeEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Metadata gui element</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="categoryBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>GUI order</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="orderSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="descEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
